analytics/tracker_object.rst

```diff
@@ -0,0 +1,543 @@
+.. highlight:: js
+.. default-domain:: js
+
+Tracker Object Functions
+========================
+
+This document describes all functions available for Tracker object and how to create its instance.
```

Comment:
Here Tracker is uppercase and 6 lines below its lowercase, please decide one standard.

Reply:
It should be written in uppercase, fixed